extension/src/main/scala/akkapi/cluster/ClusterWithLeds.scala

```diff
@@ -0,0 +1,23 @@
+package akkapi.cluster
+
+import akka.NotUsed
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.scaladsl.adapter.TypedActorSystemOps
+import akka.management.scaladsl.AkkaManagement
+import com.typesafe.config.ConfigFactory
+
+trait ClusterWithLeds {
+
+  def main(args: Array[String]): Unit = {
+
+    val actorSystemName = s"""pi-${ConfigFactory.load().getString("cluster-node-configuration.cluster-id")}-system"""
+
+    val system = ActorSystem[NotUsed](Behaviors.ignore, actorSystemName)
+
+    LedStatusTracker(system).start()
+
+    // Start Akka HTTP Management extension
+    AkkaManagement(system.toClassic).start()
+  }
```

Comment:
The only problem I see with this is that this freezes the functionality of the application as a whole (which is fine in the Pi-Akka-Cluster application.
Would moving the `main` method out of here and providing a method (`startLedStatusTracker`) that takes an ActorSystem as an argument solve this. This would allow a user to create a `main` method in which they'd create the `ActorSystem`  followed by an invocation of `startLedStatusTracker`. WDYT?

Reply:
Yeah, i was in two minds about whether to add this - i couldn't put it in the `core` module, though, because then there'd be a circular reference between `core` and `extension`.

I'll just remove it and put this code back into the system entry point 🤷